test/unit/user_event/clear_test.dart

```diff
@@ -43,5 +45,24 @@ void main() {
       UserEvent.clear(textarea);
       expect(textarea, hasValue(''));
     });
+
+    test('throws when error occurs in event handler', () {
```

Comment:
Also is this test redundant since we have `testEventHandlerErrors`?

Reply:
Yes it is, i was manually writing these tests before i realized i could just like make the shared test function thing...